README.md

```diff
@@ -124,7 +126,7 @@ removes the existing docker container and persistent volume before starting
 a new docker based build.
 
 -C, --configure-args <args>
-specify any custom user configuration arguments, using 
+specify any custom user configuration arguments, using
```

Comment:
apologies, I think vscode got angry at the formatting here (should disappear if you hide whitespace changes)